release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -263,9 +280,36 @@ On the "Enhancement Stats" tab, a table was created to track the Doc Stats and t
 
 This is very helpful for weekly reports and managing deadlines
 
+### Communicate major deadlines
+
+COMMUNICATE ALL 3 MAJOR DATES AT LEAST A WEEK PRIOR, INCLUDING THE RELEASE DATE REPO FREEZE, FOLLOWING THE BELOW METHODS:
+
+- Keep developers apprised of due dates for docs.
+    Send announcements to:
+    - [Kubernetes Dev Group](https://groups.google.com/forum/#!forum/kubernetes-dev)
```

Comment:
Is the announcement to k-dev necessary for Docs Lead? I think both Enhancements Lead and the Release Lead takes care of most of the announcements related to the release for k-dev. We should promote communication between the subteams and make sure important dates are communicated without needing Docs Lead to send out a separate announcement.

Reply:
I agree that the Enhancements Lead and the Release Lead take care of announcements to k-dev. I'll modify this list